style/style.css

```diff
@@ -0,0 +1,12 @@
+body {
+  height: 100%;
+}
+
+.responsive {
+  width: 50%;
+  height: 50%;
+}
+
+#img-div {
```

Comment:
This is a great job. I think you can use CSS flexbox to arrange your elements more efficiently

Reply:
- [ ] Kindly refer to my previous comment above.